test/assets/unrunnableTestCases.csv

```diff
@@ -842,6 +842,8 @@ fn-parse-json-944,AssertionError: expected [Function] to throw error matching /F
 fn-parse-json-945,AssertionError: expected [Function] to throw error matching /XPTY0004/ but got 'XPST0017: Function parse-json with arity of 2 not registered. Did you mean "Q{http://www.w3.org/2005/xpath-functions}parse-json (xs:stringnull)"?'
 fn-parse-json-946,Error: XPST0017: Function parse-json with arity of 2 not registered. Did you mean "Q{http://www.w3.org/2005/xpath-functions}parse-json (xs:stringnull)"?
 fn-parse-json-953,AssertionError: expected [Function] to throw error matching /FOJS0001/ but got 'XPST0017: Function codepoints-to-string with arity of 1 not registered. No similar functions found.'
+path014,SyntaxError: Expected "!", "!=", "*", "+", ",", "-", "/", "//", "<", "<<", "<=", "=", "=>", ">", ">=", ">>", "[", "and", "cast", "castable", "div", "eq", "except", "ge", "gt", "idiv", "instance", "intersect", "is", "le", "lt", "mod", "ne", "or", "to", "treat", "union", "|", "||", or "}" but "(" found.
+path017,Error: No selector counterpart for: computedTextConstructor.
 K-ContextPositionFunc-20,Error: No selector counterpart for: treatExpr.
```

Comment:
These two new flailing tests are to be expected, we don't do computed text constructors (yet) nor do we do whatever syntax construct is being used in the other test.